src/index.html

```diff
@@ -1,14 +1,57 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+        maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div id="home">
+        <a href="#home" class="logo">
+          <img src="./images/logo.png" alt="Logo">
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#"> Apple </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#"> Samsung </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#"> Smartphones </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#"> Gadgets </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#"> Tablets </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#"> Photo </a>
+          </li>
+          <li class="nav__item">
```

Comment:
I suggest you putting 1 blank line between sibling tags (when they have some content) to improve readability